src/ng-xform/form-layout/form-layout.component.ts

```diff
@@ -0,0 +1,62 @@
+import { FormControl, FormGroup } from '@angular/forms';
+import { Observable } from 'rxjs/Observable';
+import { Component, Input } from '@angular/core';
+
+
+
+/**
+ * Component to generate a bootstrap form field of numeric type
+ *
+ * :isEditing: Flag to control component state
+ * :isCheckbox: Flag to inform if it is a layout for a checkbox
+ * :isHorizontal: Flag to inform if it is a horizontal layout
+ * :isOptionl: Flag to append '(optional)' on the label
+ * :labelWidth: Number of columns of the bootstrap system grids to apply on the label element
+ * :label: Label of the field
+ * :formattedValue: Value to display when the field been on reading state
+ * :styleClasses: css classes to apply on the html element
+ */
+@Component({
+  selector: 'ng-xform-form-layout',
+  templateUrl: './form-layout.component.html',
+  styleUrls: [
+    'orm-layout.component.css'
+  ]
+})
+export class FormLayoutComponent {
+  @Input() isHorizontal = false;
+  @Input() isEditing = false;
+  @Input() isCheckbox = false;
```

Comment:
Change `isCheckbox` to `hideLabelOnEdit` (on the attribute docs, mentions that it's primary use is for checkboxes)